webapp/portlet/src/main/webapp/spaces-list/components/ExoSpaceCardFront.vue

```diff
@@ -237,6 +237,7 @@ export default {
     confirmMessage: '',
     okMethod: null,
     displaySecondButton: false,
+    isChatEnabled: false,
```

Comment:
We should manage Chat button in the extension.js file, and not in the space card

Reply:
The extensions are managed in a computed property which filters on all extension to check if they are enabled. In the extension.js i added an async method to the chat extension's enabled function to check if a space's chat is enabled. Since javascript's array.filter doesn't support async functions its not possible to do so.

Reply:
Please find a way to solve it.

Reply:
Done